package.json

```diff
@@ -1,196 +1,26 @@
 {
-  "name": "nabucasa_support_theme",
+  "name": "nabucasa_support_content",
   "version": "1.0.0",
   "repository": "git@github.com:nabucasa/support.git",
   "type": "module",
   "scripts": {
-    "start": "concurrently -k -r 'rollup -c -w' 'wait-on script.js style.css && npm run zendesk-theme-preview'",
-    "build": "NODE_ENV=production rollup -c",
-    "prebuild": "rm -f assets/*-bundle.js",
-    "eslint": "eslint src",
-    "prepare": "husky install",
-    "download-locales": "node ./bin/update-translations",
-    "test": "jest",
-    "i18n:extract": "node bin/extract-strings.mjs",
-    "i18n:update-translations": "node bin/update-modules-translations.mjs",
-    "zcli": "zcli",
-    "lint:all:check": "npm run lint:textlint:check && npm run lint:remark:check",
-    "lint:all:fix": "npm run lint:textlint:fix",
-    "lint:remark:check": "remark content --verbose --frail",
-    "lint:textlint:check": "textlint content",
-    "lint:textlint:fix": "textlint --fix content",
-    "zendesk-theme-preview": "export $(grep -v -E '^\\s*#|^\\s*$' .env | sed 's/\\s*#.*//' | xargs) && zcli themes:preview --bind 127.0.0.1 --port 4568",
-    "zendesk-content-preview": "cd content && eleventy --serve",
-    "zendesk-content-build": "cd content && rm -rf _site && eleventy",
-    "zendesk-content-build-preview": "cd content && rm -rf _site && PREVIEW_BUILD=1 eleventy",
-    "zendesk-content-deploy": "export $(grep -v -E '^\\s*#|^\\s*$' .env | sed 's/\\s*#.*//' | xargs) && cd content && node deploy.js",
+    "build": "npx @11ty/eleventy",
+    "build-nocolor": "cross-env NODE_DISABLE_COLORS=1 npx @11ty/eleventy",
+    "start": "npx @11ty/eleventy --serve --quiet",
+    "debug": "cross-env DEBUG=Eleventy* npx @11ty/eleventy",
+    "debugstart": "cross-env DEBUG=Eleventy* npx @11ty/eleventy --serve --quiet",
+    "benchmark": "cross-env DEBUG=Eleventy:Benchmark* npx @11ty/eleventy",
+    "zendesk-content-preview": "eleventy --serve",
+    "zendesk-content-build": "rm -rf _site && eleventy",
+    "zendesk-content-build-preview": "rm -rf _site && PREVIEW_BUILD=1 eleventy",
+    "zendesk-content-deploy": "export $(grep -v -E '^\\s*#|^\\s*$' .env | sed 's/\\s*#.*//' | xargs) && node deploy.js",
```

Comment:
All these `zendesk-content-` prefixed command should replace their standard couunterparts here as the contents of this repository moving forward is content only